src/presentation/controller/AbstractController.ts

```diff
@@ -108,13 +106,6 @@ abstract class AbstractController {
     submissionId: string,
     generalPartnerId: string
   ) {
-    // limited partnership urls that can exist with or without ids
-    const URLS = [NAME_URL];
-
-    if (transactionId && submissionId && URLS.includes(url)) {
-      url = url.replace(BASE_URL, BASE_WITH_IDS_URL);
-    }
-
```

Comment:
had to remove this name_url part as it would always replace the url with the 'ids' version if we had ids in the url. For this story we want to go from url with ids to url without ids when we have ids and this was preventing that. The routing with/without ids for name page is taken care of in teh LimitedPartnershipController.